src/Ml.Cli.WebApp/ClientApp/src/Local/Comparison/__snapshots__/index.spec.js.snap

```diff
@@ -1239,7 +1239,7 @@ exports[`Check a compare file insertion Find table result of KOs after file inpu
     </div>
   </div>
   <div
-    class="af-title-bar af-title-bar--backhome"
+    class="af-title-bar af-title-bar--backhome "
```

Comment:
Idem?

Reply:
That was due to the acceptance of classmodifiers for the titlebar. I updated it !